### PR TITLE
feat: support custom output directory via -o / --output-dir

### LINKED ISSUE #60: Support specifying output directory
It'd be nice if it were possible to specify the output directory to use instead of [generating it on the fly using the `PID`](https://github.com/davidmarkclements/0x/blob/4074aa2021adaef208d9fc9fd239e8e68f38df3a/index.js#L37).

Especially in a docker world where specific files and directories are mounted into containers, having non-deterministic paths can be a pain.

Something like `-O / --output-dir` would be awesome 🙂

Thanks for putting out such a quality tool!

### PR DESCRIPTION
**Pull request by knksmith57**
resolves #60

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,13 +34,29 @@ module.exports = function (args, binary) {
...
 
   if (args['timestamp-profiles']) {
     name += '-' + Date.now()
   }
 
-  return name
+  return path.resolve(process.cwd(), name)
+}
+
+function makeDirectoryIfNotExistsSync(path) {
```
Comment (davidmarkclements) [inline index.js]: `ensureDirExists`

or 

`mkdirp`

**Comment by davidmarkclements** [comment]
Hey thanks for the PR! This is a useful feature, just a small change req and then we can merge 👌

File: index.js (hunk 1 above)
Comment (davidmarkclements) [inline index.js]: Actually the former since mkdirp is really about creating nested structures

**Comment by knksmith57** [comment]
Awesome, thanks @davidmarkclements. PR feedback addressed

**Pull request merged**